features/refunding_order_shipping_cost.feature

```diff
@@ -0,0 +1,32 @@
+@refunds
+Feature: Refunding an order shipping cost
+    In order to give back money spent by Customer for order shipment
+    As an Administrator
+    I want to be able to refund an order shipping cost
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10.00"
+        And the store has "Galaxy Post" shipping method with "$20.00" fee
+        And the store allows paying with "Space money"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought 2 "Mr. Meeseeks T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And the order "#00000022" is already paid
+        And I am logged in as an administrator
+
+    @ui @application
```

Comment:
Could we have tags in the first line of this file?

Reply:
To be honest, I really hate how scenarios look with no tags above them :/ And there are fewer things to change if some only-ui or only-application scenario would be added to this file in the future

Reply:
😢